walk-in-interview/src/main/java/com/google/job/servlets/JobsListingsServlet.java

```diff
@@ -0,0 +1,229 @@
+package com.google.job.servlets;
+
+import com.google.job.data.*;
+import com.google.gson.Gson;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+import java.sql.Time;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.*;
+
+/**
+ * Servlet that handles getting the job listings given the filters.
+ */
+@WebServlet("/jobs/listings")
+public final class JobsListingsServlet extends HttpServlet {
+    private static final long TIMEOUT = 5;
+
+    public static final String MIN_LIMIT_PARAM = "minLimit";
+    public static final String MAX_LIMIT_PARAM = "maxLimit";
+    public static final String REGION_PARAM = "region";
+    public static final String SORT_BY_PARAM = "sortBy";
+    public static final String ORDER_PARAM = "order";
+    public static final String PAGE_SIZE_PARAM = "pageSize";
+    public static final String PAGE_INDEX_PARAM = "pageIndex";
```

Comment:
why these need to be public? seems i did not find other places except this servlet using these constants?